src/app/shared/components/template/services/instance/template-action.service.ts

```diff
@@ -222,6 +222,9 @@ export class TemplateActionService extends TemplateInstanceService {
         if (emit_value === "set_skin") {
           this.skinService.setSkin(args[1]);
         }
+        if (emit_value === "set_skin_only") {
```

Comment:
nit(blocking)
I don't really like this syntax, if it really is an override probably using additional args on set_skin is a better alternative (e.g. `emit: set_skin:my_skin_name | preserve_theme` or something to that effect). 

But even still that doesn't make much sense to me, if a skin defines a theme it should probably be applied (particularly if you are trying to preserve a theme which isn't officially supported by the skin) - if you really want a skin that doesn't change theme then you can just create a different skin without a default theme. So if you agree I think this can go

Reply:
OK, I agree. 

I was imagining the scenario of two skins that have different default themes but share an available (and currently active) theme. But I hadn't considered the idea that a skin doesn't need to specify a default skin – it makes sense that if it does then that theme should probably be applied along with the skin.

And you're right that there are ways around it that are probably more appropriate. Another way to override a skin's default theme would be to explicitly call the `set_theme` action from the template. Getting rid of this option cleans up the code in the skin service too.